orbit/cmd/orbit/orbit.go

```diff
@@ -614,11 +624,16 @@ func main() {
 		}
 
 		var (
-			options              []osquery.Option
+			options []osquery.Option
+			// optionsAfterFlagfile is populated with options that will be set after the '--flagfile' argument
+			// to not allow users to change their values on their flagfiles.
```

Comment:
Not sure I understand this comment.

Reply:
We want to set `--database_path=foobar` after `--flagfile osquery.flags` so that users cannot override `--database_path` on their `osquery.flags`.